data/json/items/tool_armor.json

```diff
@@ -2211,6 +2211,51 @@
     },
     {
         "type" : "TOOL_ARMOR",
+        "id" : "bagpipes",
+        "name" : "bagpipes",
+        "name_plural": "bagpipes",
+        "category" : "tools",
+        "weight" : 2300,
+        "color" : "green",
+        "covers" : ["TORSO"],
```

Comment:
Hmm.  Last I heard, keeping one's pipes inflated was not trivial, and would probably be worth 20 or so mouth encumbrance/significantly higher-than-average stamina drain.

And fuck yeah bagpipes.  :-)